idealingua/idealingua-runtime-rpc-scala/src/main/scala/com/github/pshirshov/izumi/idealingua/runtime/rpc/IRTCirceMarshaller.scala

```diff
@@ -4,16 +4,18 @@ import io.circe.{DecodingFailure, Json}
 
 import scala.language.higherKinds
 
-trait IRTCirceMarshaller[R[_, _]] extends IRTResultTransZio[R] {
+abstract class IRTCirceMarshaller[R[_, _] : IRTResult] {
```

Comment:
Are you sure it has to use `R` context? IRTCirceMarshaller/IRTWrappedClient can be made parameterless, so they can remain `objects`